triple-store/triple-store-impl-rdf4j/src/main/java/com/powsybl/triplestore/impl/rdf4j/TripleStoreFactoryServiceRDF4J.java

```diff
@@ -22,9 +22,17 @@ public TripleStore create() {
         return new TripleStoreRDF4J();
     }
 
+    @Override
+    public TripleStore copy(TripleStore source) {
+        // FIXME use a copy constructor?
```

Comment:
If it exists, you should use it

Reply:
for simplicity, we will rely on an `add(Triplestore source)` method, that can be required through the interface